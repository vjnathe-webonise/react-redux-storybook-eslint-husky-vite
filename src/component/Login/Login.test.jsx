import renderer from 'react-test-renderer';
import Login from './Login';
import { BrowserRouter } from 'react-router-dom';

it('Login renders correctly', () => {
  const tree = renderer
    .create(
      <BrowserRouter>
        <Login />
      </BrowserRouter>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
