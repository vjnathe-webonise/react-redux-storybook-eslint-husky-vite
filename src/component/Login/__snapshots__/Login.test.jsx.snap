// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Login renders correctly 1`] = `
<div>
  <h1>
    Login
  </h1>
  <a
    href="/"
    onClick={[Function]}
  >
    Dashboard
  </a>
</div>
`;
